### DISCUSSION TITLE
confuzing about consult-find on dired mode

### DISCUSSION THREAD

**Discussion by sincebyte** [minad/consult#851] [Q&A] [answered]
why in the dired mode use consult-find and choose one , but create a file at other buffer relative location ? 
the step is 
1. open buffer A locate at /projectA
2. use recent-file and choose a dir locate at /projectB/, and now I'm in the dired mode
3. use consult-find /projectB/b.java
4. the expect is open /projectB/b.java. but create a /projectA/b.java

**Comment by sincebyte** [comment]
hi, any body could help me?

**Reply by minad** [reply]
Can you please create a minimal recipe which demonstrates this, starting from emacs -Q? See https://github.com/minad/consult#bug-reports.

**Comment by sincebyte** [comment]
It's really difficult for me . I'm using doom emacs. 
I think doom emacs lastest default setting will reappear this issue.

**Reply by minad** [reply, marked as answer]
I recommend to ask on the Doom Discord about Doom-specific problems.

**Reply by oantolin** [reply]
The Doom Discord sounds like an apocalyptic disagreement.
